framework/content/ecl-typography/ecl-typography-lists/_lists.scss

```diff
@@ -1,22 +1,7 @@
-ul:not(.listing, .social-media-links__list, .file__translations-list, .site-switcher__list, .menu, .site-user-menu__list, #admin-menu-wrapper ul),
-ol {
-  max-width: 32em;
```

Comment:
I can add this 32em rule to our list classes, but the idea was not to override default html styles.